GliaWidgets/Sources/ViewController/Common/Alert/AlertViewController.swift

```diff
@@ -27,10 +27,29 @@ class AlertViewController: UIViewController {
             accepted: () -> Void,
             declined: () -> Void
         )
+
+        /// Indicating presentation priority of an alert.
+        /// Based on comparing values we can decide whether an alert can be replaced with another alert.
+        fileprivate var presentationPriority: PresentationPriority {
```

Comment:
`fileprivate` and `private` work the same way in one file, so we can simplify it and use just `private` instead. would it make sense?

Reply:
When `presentationPriority` is `private`, it can't be accessible out of `Kind` scope. So that's why I make it `fileprivate`

Reply:
oh, that is in nested class. ok.